resources/settings.xml

```diff
@@ -3,6 +3,9 @@
     <setting id="search_signed" label="30110" type="bool" default="false" />
     <setting id="search_ad" label="30120" type="bool" default="false" />
     <setting id="subtitles" label="30130" type="bool" default="false" />
+    <setting label="Enable BBC iD" default="false" type="bool" id="bbc_id_enabled"/>
+    <setting label="BBC Username" default="" type="text" id="bbc_id_username" enable="eq(-1,true)" subsetting="true"/>
+    <setting label="BBC Password" default="" type="text" id="bbc_id_password" enable="eq(-2,true) + !eq(-1,)" subsetting="true" option="hidden"/>
```

Comment:
The settings.xml should be localized. Please move the strings to resource/language/English/strings.xml and replace labels in settings.xml with IDs.